sdk/src/transaction/sanitized.rs

```diff
@@ -76,6 +76,21 @@ impl SanitizedTransaction {
         })
     }
 
+    pub fn try_from_legacy_transaction(tx: Transaction) -> Result<Self> {
+        tx.sanitize()?;
+
+        if tx.message.has_duplicates() {
+            Result::<Self>::Err(TransactionError::AccountLoadedTwice).unwrap();
+        }
```

Comment:
This can probably get refactored so `from_transaction_for_tests` calls this, especially if this line becomes `return Err(TransactionError::AccountLoadedTwice)`

Reply:
Oh 💯 true! Flitted through this one too quickly